assets/services/parachute/test_parachute.py

```diff
@@ -0,0 +1,298 @@
+import os
+import unittest
+
+from pymavlink import mavutil
+from pymavlink.dialects.v20 import common as mavlink
+
+
+class MavlinkUDPConnection:
```

Comment:
this is redefined in 2 different places. Should consolidate

Reply:
the original idea here was so that the test and the parachute emulator don't have dependency and can run independently without needing the other file